pkg/commands/workload.go

```diff
@@ -228,6 +261,12 @@ func (opts *WorkloadOptions) LoadDefaults(c *cli.Config) {
 	opts.ExcludePathFile = c.TanzuIgnoreFile
 }
 
+func ErrMultipleSourceOf(names ...string) validation.FieldErrors {
```

Comment:
I know that this is something specific for workload, but we have some other files /folders  where we record the actual error types, so I'd move this there.

Reply:
Also, should this have its own test function?

Reply:
I have moved the function under validations and added unit test cases